### DISCUSSION TITLE
giscus comment's don't show up on posts/pages.

### DISCUSSION THREAD

**Discussion by ColtonKawamura** [alshedivat/al-folio#1865] [Q&A] [answered]
I have enabled giscus and discussions for my repository, but comments still do not show up after signing in to GitHub and submitting the comment.

**Comment by alshedivat** [comment]
can you try changing the category name in your `_config.yml` [here](https://github.com/ColtonKawamura/coltonkawamura/blob/master/_config.yml#L141) from `Comments` to `comments` to match the lower case of the category name that you have in discussions in your repo?

**Reply by ColtonKawamura** [reply]
Will try right now and report back.

Thank you for your reply!

**Reply by ColtonKawamura** [reply]
No change. Made suggested change and also added my repo "coltonkawamura/coltonkawamura" to line above with no change.

Reactions also will not populate

**Reply by alshedivat** [reply]
hmm, surprised that it doesn't work. try adding a couple more fields to your config under `giscus`:
```YAML
giscus:
  # all other fields
  data-repo-id: R_kgDOKjP3YQ
  data-category-id: DIC_kwDOKjP3Yc4CaV1i
```

**Reply by ColtonKawamura** [reply]
Added. No change still. I'm happy to re-template the al-folio repository to try with a fresh start and try to trouble shoot from there if that helps.

**Reply by ColtonKawamura** [reply]
For more information: by using the script provided in the  _Enable giscus_  section of [the giscus app page](https://giscus.app), within project.md provides a solution.

Still not sure how to solve with the current al-folio structure

**Reply by alshedivat** [reply, marked as answer]
note: you may need to enable giscus app for your repo: https://github.com/apps/giscus
